test/ui-test/src/screens/StatusCommunityScreen.py

```diff
@@ -146,12 +149,28 @@ def create_community_category(self, community_category_name, community_channel_n
             test.fail("Can't find channel " + community_channel_name)
 
         click_obj_by_name(CreateOrEditCommunityCategoryPopup.COMMUNITY_CATEGORY_BUTTON.value)
+        
+    def delete_community_category(self, community_category_name):
+        [loaded, category] = self._find_category_in_chat(community_category_name)
+        if not loaded:
+            test.fail("Can't find category " + community_category_name)
+
+        # For some reason it clicks on a first channel in category instead of category
+        squish.mouseClick(category.parent, squish.Qt.RightButton)
+        click_obj_by_name(CommunityScreenComponents.COMMUNITY_DELETE_CATEGORY_MENU_ITEM.value)
+        click_obj_by_name(CommunityScreenComponents.COMMUNITY_CONFIRM_DELETE_CATEGORY_BUTTON.value)
+        time.sleep(0.5)
```

Comment:
Why is this needed?

Reply:
to ensure that categories list is updated, on the next step we trying to find deleted category

Reply:
removed